rs/pocket_ic_server/src/pocket_ic.rs

```diff
@@ -356,9 +357,10 @@ impl PocketIc {
             hypervisor_config.max_query_call_graph_instructions = instruction_limit;
         }
         // bound PocketIc resource consumption
-        hypervisor_config.embedders_config.min_sandbox_count = 0;
         hypervisor_config.embedders_config.max_sandbox_count = 64;
         hypervisor_config.embedders_config.max_sandbox_idle_time = Duration::from_secs(30);
+        hypervisor_config.embedders_config.max_sandboxes_rss_size =
+            NumBytes::new(2 * 1024 * 1024 * 1024);
```

Comment:
I'm not sure if we need to impose a lower limit for pocket IC. If no one is complaining about it's memory usage the 64 count should still be fine, right?

Reply:
Discussed offline, decided to go as is.